sphinx_antsibull_ext/assets.py

```diff
@@ -0,0 +1,44 @@
+# coding: utf-8
+# Author: Felix Fontein <felix@fontein.de>
+# License: GPLv3+
+# Copyright: Ansible Project, 2020
+'''
+Handling assets.
+'''
+
+from __future__ import (absolute_import, division, print_function)
+
+
+import os
+
+from pkg_resources import resource_filename
```

Comment:
Is there anyway to avoid pkg_resources?  There's a backport of importlib.resources, for instance: https://pypi.org/project/importlib-resources/

(pkg_resources can cause large import time slowdowns.  I do whatever I can to avoid it.)

Reply:
Removed in af858f012a95d4b524cc8a33431ad6f4beb5713a.